mainapp/templates/login.html

```diff
@@ -21,6 +20,13 @@ <h2>Login</h2>
                         required>
                 </div>
                 <input type="submit" class="btn btn-primary" value="Login">
+                <a>&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;</a>
```

Comment:
Could you explain what `&emsp` does and why we need them? I guess it's for the blank character?

Reply:
Hey, yes they are for space. To place the button at the correct position.

Reply:
instead of using all of these `&emp;` spaces, it seems like it would be much cleaner to just use CSS to weight the button to the side, or enclose the login and back buttons in a another div and then use CSS on that. bootstrap has a lot of options for positioning elements with CSS and `<div>`s